specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -1593,7 +1593,7 @@
       ],
       "x-ms-enum": {
         "name": "AnalyzerName",
-        "modelAsString": false
+        "modelAsString": true
```

Comment:
Is the idea to add the names & descriptions once AutoRest is fixed?

Reply:
Yup :) More precisely, I plan on adding them once we are ready to remove the SDK customization!